public/sample.html

```diff
@@ -103,6 +89,37 @@ <h2>Payment</h2>
                 </select>
                 <input type="submit" value="Initiate" />
                 <button id="signature">Get Signature</button>
+
+                <div>
+                    <label for="xAmount"><strong>Amount</strong></label>
+                    <input type="text" name="xAmount" value="1.99" />
+
+                    <label for="xTax"><strong>Tax</strong></label>
+                    <input type="text" name="xTax" value="0.04" />
+
+                    <label for="xInvoice"><strong>Invoice</strong></label>
+                    <input type="text" name="xInvoice" value="12345" />
+
+                    <label for="xStoreId"><strong>Store ID</strong></label>
+                    <input type="text" maxlength="4" name="xStoreId" value="1" />
+
+                    <!-- <label for="xStreet"><strong>Street</strong></label>
+                    <input type="text" name="xStreet" value="123 Main St" /> -->
+                    <!-- <label for="xZip"><strong>ZIP</strong></label>
+                    <br />
+                    <input type="num" name="xZip" value="11111" /> -->
```

Comment:
Commented on purpose?

Reply:
Yes, I couldn't see it showing up on gateway rpt. Is the point of this only for semi-integrated? Or is this missing from my device config?

Reply:
can you try a keyed transaction with street/zip set?

Reply:
can you try a keyed transaction with street/zip set?

Reply:
Keyed on the pax? We don't support that.